r2/r2/controllers/front.py

```diff
@@ -874,14 +879,22 @@ def GET_about(self):
         return Reddit(content=item).render()
 
     @require_oauth2_scope("read")
-    @api_doc(api_section.subreddits, uses_site=True)
+    @api_doc(api_section.subreddits, uri='/r/{subreddit}/about/sidebar')
     def GET_sidebar(self):
         """Get the sidebar for the current subreddit"""
-        usertext = UserText(c.site, c.site.description)
-        return Reddit(content=usertext).render()
+        if isinstance(c.site, FakeSubreddit):
+            return self.abort404()
+        if is_api():
```

Comment:
Have you run across [`config/templates.py`](https://github.com/13steinj/reddit/blob/13steinj-api-doc/r2/r2/config/templates.py)?  It allows overriding the template separate from the controller.  I'm not sure if it's a good fit here, since we're just using a nice generic `UserText`, but it would be nice to not have the entire body of the function split up this way.